### ISSUE TITLE
Renaming files with multiple artists

### ISSUE THREAD

**Issue by radusuciu** [nicfit/eyeD3#149]
Hi. According to the ID3 spec 

> The 'Lead artist(s)/Lead performer(s)/Soloist(s)/Performing group' is used for the main artist(s). They are seperated with the "/" character.

If I do this:

```python
from eyed3.id3.tag import TagTemplate
import eyed3

audio_file = eyed3.load('test.mp3')
# clear any existing tags
audio_file.initTag()

tag = audio_file.tag
tag.artist = 'A/B'
tag.title = 'foobar'

new_name = TagTemplate('$artist - $title').substitute(tag, zeropad=True) # A-B - foobar

audio_file.rename(new_name)
```
I end up with: `A-B - foobar.mp3`. The artists are set properly in the tag, and are correctly listed as individuals. The '/' characters are getting replaced so as to not cause path issues and while that's a good general fix, I think the relatively common case of multiple artists warrants independent handling.

In particular, I would like to see this being the result of the above code: `A, B - foobar`, with a comma delimiter between artists.

Thanks for the awesome library and tool!

**Comment by nicfit** [comment]
The semantics of TagTemplate's `path_friendly` keyword arg will change in eyeD3 0.8.6, and will help here.  Currently `path_friendly=True` is the default, and when set then "-" is used, as you've shown. For the next version the default will be "-" to achieve the same effect, but it now allows what you want.

```python
new_name = TagTemplate('$artist - $title', path_friendly=", ").substitute(tag, zeropad=True)
```

**Comment by nicfit** [comment]
Fixed in  6be581d

**Comment by radusuciu** [comment]
Thank you!
